icu4j/main/classes/core/src/com/ibm/icu/impl/CaseMapImpl.java

```diff
@@ -733,25 +739,62 @@ public static String toTitle(int caseLocale, int options, BreakIterator iter, Ch
 
                         // Special case Dutch IJ titlecasing
```

Comment:
Done, although I also added a `else if (~c == 'I' || ~c == 'Í')` condition because when the input started with a capital 'I', the integer value `c` seemed to be the bitwise-negation of the code point integer value.